.github/workflows/operator-ci.yml

```diff
@@ -36,4 +36,35 @@ jobs:
 
       - name: Build the Keycloak Operator
         run: |
-          mvn clean package -nsu -B -e -pl operator -Doperator -Dquarkus.container-image.build=true -Dquarkus.kubernetes.deployment-target=minikube
+          mvn clean package -nsu -B -e -pl operator -Doperator -Dquarkus.container-image.build=true -Dquarkus.kubernetes.deployment-target=minikube -DskipTests
```

Comment:
if we are already running the test I think that we can completely remove this line.

Reply:
:+1:

Reply:
the line and the step I would say.
In fact if the e2e test passes we should generate the snapshot image and deploy it on quay

Reply:
yup, perfect 👍